app/views/comments/new.html.erb

```diff
@@ -1,9 +1,17 @@
-<h1>New comment</h1>
+<a href="<%= user_post_path(@user, @post) %>">
+  <button id="back-btn">⬅</button>
+</a>
+<h1>Create a New Comment</h1>
+<h2>Post <%= @post.id %></h2>
+<h2>Current User <%= @user.name %></h2>
 
-<%= render "form", comment: @comment %>
+<%= form_with model: @comment, url: user_post_comments_path(@user, @post), local: true do |form| %>
```

Comment:
- [ ] When I try to create a comment using this form, it is not added to the user. On clicking the submit button, nothing is added to the database and their is no redirection. Please fix this 
![image](https://github.com/AbdelazizSoliman/Blog-App/assets/45460192/0b6f60cd-3f5a-4664-9be8-f8093126039b)

Reply:
Still trying to fix.

Reply:
Solved!